lib/supported_devices.lua

```diff
@@ -32,6 +32,10 @@ local supported_devices = {
     { midi_base_name= 'launchpad pro mk3',      device_type='launchpad_rgb' },
     { midi_base_name= 'launchpad x 2',          device_type='launchpad_x' },
     { midi_base_name= 'launchpad x 2 2',        device_type='launchpad_x' },
+    { midi_base_name= 'launchpad Pro 1',        device_type='launchpad_rgb' },
+    { midi_base_name= 'launchpad Pro 2',        device_type='launchpad_rgb' },
+    { midi_base_name= 'launchpad Pro 3',        device_type='launchpad_rgb' },
```

Comment:
So these devices seem to create 3 USB interfaces, 1. is normally the proprietary interface, 2. is the general midi interface and 3 is something else? In short I think you only need to add 'launchpad pro 2'. Could you please try this modification and confirm if it works?

Thanks and sorry for the very late response.